ca_driver/src/create_driver.cpp

```diff
@@ -222,6 +227,11 @@ void CreateDriver::undockCallback(const std_msgs::EmptyConstPtr& msg)
   robot_->setMode(create::MODE_FULL);
 }
 
+void CreateDriver::mainMotorCallback(const std_msgs::Float32ConstPtr& msg)
+{
+  robot_->setMainMotor(msg->data);
```

Comment:
Please guard against invalid values (ie. outside the range [-1.0, 1.0]) and output a `ROS_ERROR` accordingly.

Reply:
@jacobperron How do you get the param to be in range [-1,1]? You can't convert Float32ConstPtr& to Float 32